frontend/javascripts/oxalis/model/sagas/proofread_saga.ts

```diff
@@ -287,8 +287,10 @@ function* splitOrMergeOrMinCutAgglomerate(
   }
   const { layerName, agglomerateFileMag, getDataValue } = preparation;
 
-  const sourceNodePosition = sourceTree.nodes.get(sourceNodeId).position;
-  const targetNodePosition = targetTree.nodes.get(targetNodeId).position;
+  // Use untransformedPosition because agglomerate trees should not have
+  // any transforms, anyway.
```

Comment:
I'm not sure how future-proof this assumption is. I'd rather assert this explicitly or at least show a warning and log an error to airbrake.

Reply:
I added a check for this which cancels proofreading and notifies the user. I didn't add the airbrake logging, because I think, that users will tell us if they really need this feature.